index.html

```diff
@@ -4567,7 +4567,16 @@
     <section id="rtcrtpcodeccapability*">
       <h3>dictionary RTCRtpCodecCapability</h3>
       <p>The <dfn><code>RTCRtpCodecCapability</code></dfn> dictionary provides
-      information on the capabilities of a codec.</p>
+        information on the capabilities of a codec. Exactly one <code>RTCRtpCodecCapability</code>
+        will be present for each supported combination of parameters that must be identical
+        between the offer and answer. For example:</p>
+      <ol>
+        <li>Exactly one "opus" codec, with <code>numChannels</code> set to 2,
```

Comment:
Just to make sure I understand: In SDP, opus always has 2 channels on the "rtpmap" line, but may have "stereo=0" or "stereo=1" on the "fmtp" line. But in ORTC, there's no "stereo" attribute, and it simply uses `numChannels` instead? That's certainly better from an API perspective; I just want to make sure I understand correctly.